lib_xua/lib_build_info.cmake

```diff
@@ -27,7 +27,7 @@ set(LIB_DEPENDENT_MODULES "lib_adat(1.2.0)"
                           "lib_spdif(6.1.0)"
                           "lib_sw_pll(2.2.0)"
                           "lib_xassert(4.2.0)"
-                          "lib_xud(2.3.1)")
+                          "lib_xud(2.3.2)")
```

Comment:
I've does the pin to the new 2.3.2 release

Reply:
thanks!